benchmark/exhaustive-processor/processor/src/main/kotlin/ExhaustiveProcessor.kt

```diff
@@ -0,0 +1,186 @@
+import com.google.devtools.ksp.isLocal
+import com.google.devtools.ksp.processing.CodeGenerator
+import com.google.devtools.ksp.processing.Resolver
+import com.google.devtools.ksp.processing.SymbolProcessor
+import com.google.devtools.ksp.processing.SymbolProcessorEnvironment
+import com.google.devtools.ksp.processing.SymbolProcessorProvider
+import com.google.devtools.ksp.symbol.*
+import com.google.devtools.ksp.visitor.KSTopDownVisitor
+import java.io.File
+
+class ExhaustiveProcessor(
```

Comment:
Have you checked whether this ExhaustiveProcessor uses all the newly added APIs? Except those added in Resolver or utils.